checkpoint.rst

```diff
@@ -210,14 +199,28 @@ our application's state:
     INFO:    instance started successfully
 
 
-And now we can verify the variable on the server has been properly restored to
-a value of ``1``, instead of the default of ``0``:
+And now when we get access to the application again, the request count value is ``3`` as expected,
+meaning that previous request count value is ``2``.
```

Comment:
Change `is` to `was` and either add `the` before `previous` or change `previous` to `previously the`.